app/models/service.rb

```diff
@@ -231,7 +231,7 @@ def current_effective_pricing_map
   def effective_pricing_map_for_date(date=Date.today)
     raise ArgumentError, "Service has no pricing maps" if self.pricing_maps.empty?
 
```

Comment:
Through testing, we found that it was taking longer to hit the database for these than to eager_load pricing maps beforehand and do a ruby enubrable#select to do the same thing.